Description: Remove variable output from INFO_BIN to satisfy Debian reproducible builds requirements. For details see https://reproducible.debian.net/dbd/unstable/amd64/mariadb-10.0_10.0.19-1.debbindiff.html
Author: Otto Kekäläinen <otto@seravo.fi>
Last-Update: 2015-06-21

--- a/cmake/info_macros.cmake.in
+++ b/cmake/info_macros.cmake.in
@@ -118,13 +118,12 @@ MACRO(CREATE_INFO_BIN)
     SET(TMP_DATE "(no date command known for this platform)")
   ENDIF()
   SITE_NAME(HOSTNAME)
-  FILE(APPEND ${INFO_BIN} "Build was run at ${TMP_DATE} on host '${HOSTNAME}'\n\n")
 
   # According to the cmake docs, these variables should always be set.
   # However, they are empty in my tests, using cmake 2.6.4 on Linux, various Unix, and Windows.
   # Still, include this code, so we will profit if a build environment does provide that info.
   IF(CMAKE_HOST_SYSTEM)
-    FILE(APPEND ${INFO_BIN} "Build was done on  ${CMAKE_HOST_SYSTEM} using ${CMAKE_HOST_SYSTEM_PROCESSOR}\n")
+    FILE(APPEND ${INFO_BIN} "Build was using ${CMAKE_HOST_SYSTEM_PROCESSOR}\n")
   ENDIF()
   IF(CMAKE_CROSSCOMPILING)
     FILE(APPEND ${INFO_BIN} "Build was done for ${CMAKE_SYSTEM} using ${CMAKE_SYSTEM_PROCESSOR}\n")
